automatic/libreoffice-streams/README.md

```diff
@@ -6,3 +6,5 @@ LibreOffice is the free power-packed Open Source personal productivity suite for
 
 - This package installs LibreOffice Fresh which is the stable version of LibreOffice with the most recent bleeding-edge features. This version is recommended for technology enthusiasts, early adopters or power users. For more conservative users or for deployments in enterprise or corporate environments, it is recommended to install [libreoffice-still](/packages/libreoffice-still) instead.
 - Older versions of this package are likely broken because the LibreOffice team removes the download links for older versions after each update.
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to. **
```

Comment:
```suggestion
**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Otherwise it won't show up correctly as bold text.